renovate.json

```diff
@@ -26,6 +26,12 @@
         }
     ],
     "packageRules": [
+        {
+            "matchPackageNames": [
+                "jenkinsciinfra/builder"
```

Comment:
is renovate going to be able to update that? we'll probably remove the builder from the repo at some point anyway and make it the default for infra.ci so it doesn't need to be here

Reply:
it *Should* based on https://github.com/jenkins-infra/plugin-site/blob/master/renovate.json#L14-L27 but i haven't yet been able to figure out how to get it to test the configs.

Reply:
o that's cool